modules/cluster-logging-json-log-forwarding.adoc

```diff
@@ -0,0 +1,48 @@
+[id="cluster-logging-json-log-forwarding_{context}"]
+= JSON log forwarding
+
+Forwarding JSON logs gives fluentd, Elasticsearch, and other JSON-aware log collectors direct access to JSON log entries as JSON objects.
+
+To illustrate how this works, suppose that you have the following structured JSON log entry.
+
+.Example structured JSON log entry
```

Comment:
Not sure about naming for all examples in this module: should there be a colon or "Example of the..." maybe?
Please, ignore me if it's about consistency with other chapters.

Reply:
Interesting question, but I think I'll leave this unchanged for now. `.Example` shows up 69 times in the logging docs versus 13 times for `.Sample`.  Example seems more hypothetical. Sample seems like it should show an actual sample from something real. This would be an interesting low priority item to review and tweak in the future.

Reply:
Interesting nuances coming from a native speaker are always priceless, thank you.
I agree, this is really a low priority question, we can't polish every single word forever, we'll never get work done.